Index: robotframework-3.0.2/src/bin/pybot
===================================================================
--- robotframework-3.0.2.orig/src/bin/pybot
+++ robotframework-3.0.2/src/bin/pybot
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import sys
 from robot import run_cli
Index: robotframework-3.0.2/src/bin/rebot
===================================================================
--- robotframework-3.0.2.orig/src/bin/rebot
+++ robotframework-3.0.2/src/bin/rebot
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import sys
 from robot import rebot_cli
Index: robotframework-3.0.2/src/bin/robot
===================================================================
--- robotframework-3.0.2.orig/src/bin/robot
+++ robotframework-3.0.2/src/bin/robot
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import sys
 from robot import run_cli
