Upstream-Status: Inappropriate

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 050c539..1216f28 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -15,7 +15,7 @@ endif()
 
 if(UNIX)  # LINUX, FREE_BSD, APPLE
     if (NOT APPLE)
-      set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS} -s")  # --strip-unneeded for packaging
+      set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS}")  # --strip-unneeded for packaging
     endif()
     list(APPEND PROJECT_NAMES pcm-sensor-server)
     list(APPEND PROJECT_NAMES pcm-sensor)
