Index: rpm-5.4.15/configure.ac
===================================================================
--- rpm-5.4.15.orig/configure.ac	2016-07-06 17:29:49.911817555 +0300
+++ rpm-5.4.15/configure.ac	2016-07-06 17:40:27.731816208 +0300
@@ -2075,10 +2075,10 @@
 ]])],
 [RPM_CHECK_LIB(
     [Berkeley-DB], [db],
-    [db-6.0], [db_create], [db.h],
+    [db-6.2], [db_create], [db.h],
     [yes,external], [db6],
     [ DBLIBSRCS="$DBLIBSRCS db3.c"
-      DBXY=db60
+      DBXY=db62
       AM_CONDITIONAL(WITH_DB, [ true ])
       AM_CONDITIONAL(WITH_DB_INTERNAL, [ test ".$RPM_CHECK_LIB_LOCATION" = .internal ])
       if test ".$RPM_CHECK_LIB_LOCATION" = .internal; then
@@ -2111,7 +2111,7 @@
     ])
 ])
 
-if test ".$ac_cv_lib_db_6_0_db_create" != .yes -a ".$ac_cv_lib_db_5_3_db_create" != .yes; then
+if test ".$ac_cv_lib_db_6_2_db_create" != .yes -a ".$ac_cv_lib_db_5_3_db_create" != .yes; then
     CPPFLAGS="${CPPFLAGS_save}"
 fi
 
@@ -2123,11 +2123,11 @@
     [ DBLIBSRCS="$DBLIBSRCS sqlite.c" ],
     [])
 
-dnl # Sqlite 3.8.3.1 from db-6.0.30
-if test ".$ac_cv_lib_db_6_0_db_create" = .yes; then
+dnl # Sqlite 3.8.3.1 from db-6.2.23
+if test ".$ac_cv_lib_db_6_2_db_create" = .yes; then
 RPM_CHECK_LIB(
     [Berkeley-DB (+SQLite3)], [dbsql],
-    [db_sql-6.0], [sqlite3_open], [dbsql.h],
+    [db_sql-6.2], [sqlite3_open], [dbsql.h],
     [yes,external], [db6/sql],
     [
       AM_CONDITIONAL(WITH_DBSQL, [ true ])
