Index: linux/kernel/irq/manage.c
===================================================================
--- linux.orig/kernel/irq/manage.c
+++ linux/kernel/irq/manage.c
@@ -198,7 +198,7 @@ int __irq_set_affinity(unsigned int irq,
 	if (!desc)
 		return -EINVAL;
 
-	ret =  irq_set_affinity_locked(irq_desc_get_irq_data(desc), mask);
+	ret =  irq_set_affinity_locked(irq_desc_get_irq_data(desc), mask, force);
 	irq_put_desc_busunlock(desc, flags);
 	return ret;
 }
